### PR TITLE
Recompute NDVI

### PR DESCRIPTION
**Pull request by ivanzvonkov**
Recomputes NDVI when data is loaded.

Demonstration of fix

image

### CODE REVIEW

**Comment by ivanzvonkov** [comment]
Since all old models were trained on incorrect NDVI, the new NDVI values affect the model tests

**Comment by ivanzvonkov** [comment]
Will retrain all models to ensure model tests pass

**Review by bhyeh — COMMENTED** [review]
:+1:

Maybe we do #307 here to avoid potential merge conflict and otherwise just reduce that work to this PR. 

Also retraining on this branch would be dependent on merging #308 first?

**Comment by ivanzvonkov** [comment]
> 👍
> 
> Maybe we do #307 here to avoid potential merge conflict and otherwise just reduce that work to this PR.
> 
> Also retraining on this branch would be dependent on merging #308 first? Edit: neverminded, only if plan to retrain all models with GitHub actions.

Sure I can retrain the corrective labels model here. It'll take some time as it has to be done sequentially

**Comment by ivanzvonkov** [comment]
@bhyeh based on discussion this is ready for review, let me know if it looks good!

File: src/pipeline_funcs.py [hunk 1]
```diff
@@ -133,5 +133,6 @@ def run_evaluation(
 
     with (PROJECT_ROOT / DataPaths.METRICS).open("w") as f:
         json.dump(models_dict, f, ensure_ascii=False, indent=4, sort_keys=True)
+        f.write("\n")
```
Comment (ivanzvonkov) [inline src/pipeline_funcs.py:136]: Ensures the file does not have to be reformatted

**Review by bhyeh — APPROVED** [review]
> @bhyeh based on discussion this is ready for review, let me know if it looks good!

Looks good to me!

**Pull request merged**